src/annotator/annotation-sync.js

```diff
@@ -128,4 +125,8 @@ export default class AnnotationSync {
       msg: annotation,
     };
   }
+
+  destroy() {
+    this._emitter.destroy();
```

Comment:
Codecov is complaining about a missing test here. The test would be something like making sure that `AnnotationSync` does not respond to `beforeAnnotationCreated` events after `destroy` is called. Also I note that we're not disconnecting from the `Bridge` here. I guess that would be a pain to implement because the Bridge class doesn't support a way of disconnecting a specific set of callers. A workaround would be to have an internal flag in the `AnnotationSync` class that keeps track of whether `destroy` has been called.

Reply:
Typically, the responsibility of destroying an object belongs to the class that created it. In this case, `CrossFrame` creates and destroys `Bridge`.

Reply:
I have implemented the workaround you suggested in the last commit.